cpp/CMakeLists.txt

```diff
@@ -339,16 +355,17 @@ message(STATUS "CUDF_TEST_LIST set to: ${CUDF_TEST_LIST}")
 # - add google benchmark --------------------------------------------------------------------------
 
 if(BUILD_BENCHMARKS)
+    find_package(benchmark)
 
-  include(ConfigureGoogleBenchmark)
-
-  if(GBENCH_FOUND)
-    message(STATUS "Google C++ Benchmarking Framework (Google Benchmark) found in ${GBENCH_ROOT}")
-    include_directories(${GBENCH_INCLUDE_DIR})
-    add_subdirectory(${CMAKE_SOURCE_DIR}/benchmarks)
-  else()
-    message(AUTHOR_WARNING "Google C++ Benchmarking Framework (Google Benchmark) not found: automated tests are disabled.")
-  endif(GBENCH_FOUND)
+    if(benchmark_FOUND)
+        message(STATUS "Google C++ Benchmarking Framework (Google Benchmark) found in ${benchmark_ROOT}")
```

Comment:
Similarly move status message outside of the find / build if statements

Reply:
After further review, I'm unsure of the reasoning behind this if just storing in a variable. Should I just make one message that conditionally changes based off the `_FOUND` var or something? Is this for readability? The type of message changes with context. I don't quite follow.

Reply:
I think the idea is the user should get two messages:
1. Whether an existing package is found that we're going to use, or we're building the package from source ourselves
2. Where said package that we found / built is that we'll be using